build.gradle

```diff
@@ -73,6 +74,35 @@ compileTestJava {
   options.compilerArgs = ['-proc:none', '-Xlint:all','-Werror','-Xdiags:verbose']
 }
 
+model {
+  components {
+    VectorLoglessPairHMM(NativeLibrarySpec) {
+      binaries.withType(SharedLibraryBinarySpec) { binary ->
+
+        if (System.env.GATK_SKIP_NATIVE_BUILD.toString().toBoolean()) {
+          buildable = false
+        } else {
+          cppCompiler.args "-I", "${System.properties['java.home']}/../include"
+          cppCompiler.args "-I", "${System.properties['java.home']}/../include/linux"
+          cppCompiler.args "-mavx"
+          linker.args "-static-libgcc"
+
+          task copySharedLib(type: Copy) {
+            from binary.tasks
+            into "build/classes/main/org/broadinstitute/hellbender/utils/pairhmm"
```

Comment:
Is there a property that defines `build/classes/main` that could be used instead of hardcoding it?